modules/platforms/dotnet/Apache.Ignite.Tests/Sql/SqlTests.cs

```diff
@@ -759,7 +759,7 @@ public void TestExecuteBatchInvalidStatement()
             var ex = Assert.ThrowsAsync<SqlBatchException>(
                 async () => await Client.Sql.ExecuteBatchAsync(null, "select 1", [[1]]));
 
-            Assert.AreEqual("Invalid SQL statement type. Expected [DML] but got QUERY.", ex.Message);
+            Assert.AreEqual("Statement of type \"Query\" is not allowed in current context", ex.Message);
```

Comment:
The new version has less details. Can we keep "expected DML" is some form?

Reply:
The complete message looks like this:
```
Statement of type "Query" is not allowed in current context [allowedTypes=[DML]].
```

In certain cases full form is too long (because displayed name sometimes longer than name of the enum entry; i.e. display name of `TX_CONTROL` is `Transaction control statement`), so I copy-pasted only the main prefix. If you think it does make sense to compare the full form, I will update the test

Reply:
oh, I see. The full form is expected by tests, so I have no other choice...